CoreWelcomeView/test/unit/src/au/gov/asd/tac/constellation/views/welcome/WelcomeTopComponentNGTest.java

```diff
@@ -75,9 +85,34 @@ public void testCreateStyle() {
     @Test
     public void testCreateContent() {
         System.out.println("createContent");
-        final WelcomeTopComponent instance = spy(WelcomeTopComponent.class);
-        WelcomeViewPane pane = instance.createContent();
-        assertNotNull(pane);
-        assertTrue(pane.getBottomRecentSection() instanceof HBox);
+        try (final MockedStatic recentFilesStatic = Mockito.mockStatic(RecentFiles.class)) {
+            final List<RecentFiles.HistoryItem> mockList = new ArrayList<>();
+            final String path = "C:Temp";
+            final RecentFiles.HistoryItem mockItem = new RecentFiles.HistoryItem(0, path);
+            mockList.add(mockItem);
+            recentFilesStatic.when(RecentFiles::getUniqueRecentFiles).thenReturn(mockList);
+
+            final WelcomeTopComponent instance = spy(WelcomeTopComponent.class);
+            final WelcomeViewPane pane = instance.createContent();
+
+            // check WelcomeTopComponent creates WelcomeViewPane
+            assertNotNull(pane);
+            // check that the WelcomeViewPane contains the HBox for recent files
+            assertTrue(pane.getBottomRecentSection() instanceof HBox);
+
+            pane.refreshRecentFiles();
+            final Node firstChild = pane.getBottomRecentSection().getChildren().getFirst();
+            // check that the flow pane has been created
+            if (firstChild instanceof FlowPane flowPane) {
+                if (flowPane.getChildren().getFirst() instanceof Button firstButton) {
+                    // check that the first button in the recent files list is the mocked filename
+                    assertTrue(firstButton.getText() == path);
+                } else {
+                    assertFalse(true);
+                }
+            } else {
+                assertFalse(true);
```

Comment:
These asserts look odd. If the intention here is that the if condition should be true, then it would make more sense to apply the assert to the if conditions themselves